test/WebJobs.Script.Tests/Helpers/SecretsUtilityTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Helpers
+{
+    public class SecretsUtilityTest
+    {
+        [Fact]
+        public void ToKeyBytes_ReturnsExpectedValue()
```

Comment:
This is just a sanity test since I removed a Convert.FromBase64String conversion above on line 31 and we now use ToKeyBytes in all cases which handles both hex and base64.